ra_aid/agent_utils.py

```
@@ -436,45 +585,49 @@ def run_task_implementation_agent(
+        else "",
+        web_research_section=WEB_RESEARCH_PROMPT_SECTION_CHAT
+        if config.get("web_research_enabled")
+        else "",
     )
 
-    # Set up configuration
-    run_config = {
-        "configurable": {"thread_id": thread_id},
-        "recursion_limit": 100
-    }
+    config = _global_memory.get("config", {}) if not config else config
```

Ruff (0.8.2)

Use `config if config else _global_memory.get("config", {})` instead of `_global_memory.get("config", {}) if not config else config`

Replace with `config if config else _global_memory.get("config", {})`

(SIM212)